smqtk_relevancy/impls/rank_relevancy/sorted_sampling.py

```diff
@@ -0,0 +1,64 @@
+from typing import Hashable, Sequence, Tuple, Dict, Any, TypeVar, Type
+
+from numpy import ndarray
+
+from smqtk_core.configuration import (
+    from_config_dict,
+    make_default_config,
+    to_config_dict,
+)
+from smqtk_core.dict import merge_dict
+from smqtk_relevancy.interfaces.rank_relevancy import RankRelevancy, RankRelevancyWithFeedback
+
+T = TypeVar("T", bound="RankRelevancyWithSortedFeedback")
+
+
+class RankRelevancyWithSortedFeedback(RankRelevancyWithFeedback):
+    """
+    Wrap an instance of :class:`RankRelevancy` to provide feedback via
+    a sorted list of results
```

Comment:
This is not super clear to me as layman: sorted with respect to what? I read below that its being sorted by relevancy score in descending score order, but that could be made more clear here so that the reader does not have to dive into the code to know.

Reply:
Maybe these minor additions help?

```suggestion
    Wrap an instance of :class:`RankRelevancy` to provide feedback via
    a sorted, descending list of results ranked by relevancy score.
```

Reply:
LGTM!